### PR TITLE
sidebar state stored in local storage

### PR DESCRIPTION
**Pull request by avzz-19**

### CODE REVIEW

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/home-page/organization-dashboard/index.ts [hunk 1]
```diff
@@ -17,11 +17,27 @@ export interface HomePageOrganizationDashboardSignature {
...
+      storedState !== null ? JSON.parse(storedState) : true;
+  }
 
   @action
   toggleSidebar() {
     this.isSidebarCollapsed = !this.isSidebarCollapsed;
+
+    window.localStorage.setItem(
+      'sidebarState',
+      JSON.stringify(this.isSidebarCollapsed)
```
Comment (future-pirate-king) [inline app/components/home-page/organization-dashboard/index.ts:39]: keep value as `collapsed` and `expanded` based on state

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/home-page/organization-dashboard/index.ts [hunk 2]
```diff
@@ -17,11 +17,27 @@ export interface HomePageOrganizationDashboardSignature {
...
-  @tracked isSidebarCollapsed = true;
+  @tracked isSidebarCollapsed: boolean;
+
+  constructor(
+    owner: unknown,
+    args: HomePageOrganizationDashboardSignature['Args']
+  ) {
+    super(owner, args);
+
+    const storedState = window.localStorage.getItem('sidebarState');
```
[the remark this answers is not in the document]
Comment (avzz-19) [reply app/components/home-page/organization-dashboard/index.ts]: resolved internally

File: app/components/home-page/organization-dashboard/index.ts [hunk 3]
```diff
@@ -17,11 +18,28 @@ export interface HomePageOrganizationDashboardSignature {
 }
 
 export default class HomePageOrganizationDashboardComponent extends Component<HomePageOrganizationDashboardSignature> {
-  @tracked isSidebarCollapsed = true;
+  @tracked isSidebarCollapsed: boolean;
+  @service('browser/window') declare window: Window;
```
Comment (future-pirate-king) [inline app/components/home-page/organization-dashboard/index.ts:22]: move service above tracked variable and add new line to separate them

File: app/components/home-page/organization-dashboard/index.ts [hunk 4]
```diff
@@ -17,11 +18,28 @@ export interface HomePageOrganizationDashboardSignature {
...
+
+  constructor(
+    owner: unknown,
+    args: HomePageOrganizationDashboardSignature['Args']
+  ) {
+    super(owner, args);
+
+    const storedState = this.window.localStorage.getItem('sidebarState');
+    this.isSidebarCollapsed =
```
Comment (future-pirate-king) [inline app/components/home-page/organization-dashboard/index.ts:31]: add new line above

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**